<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Qualitest-PC" name="com.latch.org.testCalculator.AppTest" tests="71" failures="1" timestamp="30 Mar 2018 03:19:20 GMT" time="0.127" errors="0">
  <testcase name="TestDivideByZero2" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestAddition3" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestAddition2" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPrintRange3" time="0.001" classname="com.latch.org.testCalculator.AppTest">
    <failure type="org.testng.TestException" message="
Expected exception com.latch.org.testCalculator.CustomException but got org.testng.TestException: 
Method AppTest.TestPrintRange3()[pri:0, instance:com.latch.org.testCalculator.AppTest@2c9f9fb0] should have thrown an exception of class com.latch.org.testCalculator.CustomException">
      <![CDATA[org.testng.TestException: 
Expected exception com.latch.org.testCalculator.CustomException but got org.testng.TestException: 
Method AppTest.TestPrintRange3()[pri:0, instance:com.latch.org.testCalculator.AppTest@2c9f9fb0] should have thrown an exception of class com.latch.org.testCalculator.CustomException
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1497)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1245)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.testng.TestException: 
Method AppTest.TestPrintRange3()[pri:0, instance:com.latch.org.testCalculator.AppTest@2c9f9fb0] should have thrown an exception of class com.latch.org.testCalculator.CustomException
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1512)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:754)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	... 16 more
]]>
    </failure>
  </testcase> <!-- TestPrintRange3 -->
  <testcase name="TestPower18" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower32" time="0.019" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower8" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSquareRoot5" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower5" time="0.005" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower30" time="0.000" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower28" time="0.000" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower9" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPrintRange4" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSquareRoot1" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower26" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSutraction4" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower21" time="0.000" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSquareRoot3" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision7" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestAddition5" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMeanWithNullInputs" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower6" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSutraction1" time="0.000" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestAddition6" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower13" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision5" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower27" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSquareRoot6" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower19" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSutraction2" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision9" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower31" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestAddition1" time="0.017" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean5" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean7" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean4" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower12" time="0.006" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower1" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean11" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision4" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower17" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean6" time="0.003" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision2" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower29" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSutraction6" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestAddition4" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower3" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision8" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSquareRoot2" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision6" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision3" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSutraction5" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSutraction3" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPrintRange2" time="0.000" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivideByZero1" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower10" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean2" time="0.002" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower23" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestSquareRoot4" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower25" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestDivision1" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower24" time="0.000" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPrintRange1" time="0.003" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower4" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower15" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower16" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestMean3" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower14" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower22" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower7" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
  <testcase name="TestPower2" time="0.001" classname="com.latch.org.testCalculator.AppTest"/>
</testsuite> <!-- com.latch.org.testCalculator.AppTest -->
